src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -38,6 +38,7 @@ internal class BuildCommand : CommandBase
         };
 
         [Argument(0, Description = "The source directory.")]
+        [DirectoryExists]
```

Comment:
The question is whether this works for the cases like `.`, `c:\foo\..\dir1` etc.

Reply:
Yeah, it works with relative paths as well.

Reply:
Currently we support doing `oryx build` without supplying any value - in which case current directory is taken. This change will break it?

Reply:
it'll probably break it - i'm surprised we don't have a test for that - adding one.

Reply:
I think we do have integration tests for it

Reply:
Revert this change then?